src/browser_action.js

```diff
@@ -497,6 +497,25 @@ browseraction.createEventDiv_ = function(event) {
   return eventDiv;
 };
 
+function getGCalUrl() {
+  return "https://calendar.google.com/calendar/";
+}
+
+function isGCalUrl(url) {
+  return url.indexOf(getGCalUrl()) == 0;
+}
+
+function goToCalendar(eventUrl) {
+  chrome.tabs.getAllInWindow(undefined, function(tabs) {
+    for (var i = 0, tab; tab = tabs[i]; i++) {
+      if (tab.url && isGCalUrl(tab.url)) {
+        chrome.tabs.update(tab.id, {selected: true, url: eventUrl});
```

Comment:
What happens if the user were already in the middle of editing an event in that tab?

I think we should only switch to an existing tab if the UI displayed in that tab is the Calendar UI Home Screen, which means the URL must match more rigorously.